lib/components/TerminalView.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+class TerminalView extends PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  static get propTypes() {
+    return {
+      children: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.node),
+        PropTypes.node,
+      ]).isRequired,
+    };
+  }
+
+  render() {
+    const { children } = this.props;
+
+    /*
+     * Making the assumption that the terminal view
+     * only has 2 children
```

Comment:
I think bootstrap will still arrange relatively `smart`ly even if there are more so should still be good! could be extended/polished later to make the column widths based on how many items there are though

Reply:
I made some changes to this, will be present in additional pr